util/buildinfototags.py

```diff
@@ -0,0 +1,44 @@
+import pprint, requests, re, os
+import sys
+from xml.dom import minidom
+from optparse import OptionParser
+import json
+
+pp = pprint.PrettyPrinter(indent=4)
+
+# moar output, set = 1
+debug=0
+
+usage = "Usage: %prog [-n NAME] \n\n\
```

Comment:
you need to document usage to include the mention of curl on lines 12-14 and line 24.